src/frontend/src/pages/SettingsPage/UserSettings/UserSettings.tsx

```diff
@@ -35,9 +36,12 @@ const UserSettings: React.FC<UserSettingsProps> = ({ userId }) => {
     return <ErrorPage error={userSettings.error} message={userSettings.error.message} />;
   if (update.isError) return <ErrorPage error={update.error!} message={update.error?.message!} />;
 
-  const handleConfirm = async ({ defaultTheme }: FormInput) => {
+  const handleConfirm = async ({ defaultTheme, slackId }: FormInput) => {
     setEdit(false);
-    await update.mutateAsync({ id: userSettings.data?.id!, defaultTheme });
+    slackId === undefined
+      ? await update.mutateAsync({ id: userSettings.data?.id!, defaultTheme })
```

Comment:
it's ok to send it as undefined because that will tell prisma to change nothing on the backend. however, if the user deletes it all and tries to send empty string (aka, deleting their slack id), you should send null as the value so that prisma sets it back to nothing

Reply:
Strings only now.